src/platforms/java/common/performance/datasource.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: DataSource Instrumentation
+description: "Learn how to instrument javax.sql.DataSource"
+---
+
```

Comment:
lets add a note similar to OkHttp this is a preview https://docs.sentry.io/platforms/android/configuration/integrations/okhttp/